gtests/net/mptcp/syscalls/connect_connect.pkt

```diff
@@ -0,0 +1,31 @@
+// connect() function, connection initiated by the kernel
+// ensure that blocking connect() works as expected after unblocking one
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
+0.0         socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0        setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
++0.0        fcntl(3, F_GETFL) = 0x2 (flags O_RDWR)
++0.0        fcntl(3, F_SETFL, O_RDWR|O_NONBLOCK) = 0
+
++0.0        connect(3, ..., ...) = -1  EINPROGRESS (Operation now in progress)
++0.0          >  S   0:0(0)                    <mss 1460, sackOK, TS val 100 ecr 0,   nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
+
+
++0.0        fcntl(3, F_SETFL, O_RDWR) = 0
++0.02         <  R.  0:0(0)  ack 1  win 65535  <mss 1460, sackOK, TS val 700 ecr 100>
++0...0.020  connect(3, ..., ...) = -1 ECONNREFUSED (Software caused connection abort)
```

Comment:
it is a bit confusing:
- `ECONNREFUSED` == Connection refused
- `ECONNABORTED` == Software caused connection abort

I guess you want to use `ECONNABORTED` which is the new reported error (while before your kernel patch, it was `ECONNREFUSED`, right?)